src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tableHead = table.tHead;
+const tableBody = table.tBodies[0];
+
+tableHead.addEventListener('click', event => {
+  const target = event.target;
+
+  if (target.tagName !== 'TH') {
+    return;
+  }
+
+  const indexOfCellToSort = target.cellIndex;
+  const tableRows = [ ...tableBody.rows ];
+
+  sortTable(tableRows, indexOfCellToSort);
+});
+
+function sortTable(list, indexOfCellToSort) {
+  list.sort((previous, current) => {
+    let previousContent = previous.cells[indexOfCellToSort].textContent;
+    let currentContent = current.cells[indexOfCellToSort].textContent;
+
+    switch (indexOfCellToSort) {
+      case 2:
+        previousContent = parseInt(previousContent);
+        currentContent = parseInt(currentContent);
+
+        return previousContent - currentContent;
+      case 3:
+        previousContent = parseInt(
+          previousContent.slice(1).split(',').join('')
+        );
+
+        currentContent = parseInt(
+          currentContent.slice(1).split(',').join('')
+        );
+
+        return previousContent - currentContent;
+    }
+
+    return previousContent.localeCompare(currentContent);
```

Comment:
this should be in default case

Reply:
it's cool that you use switch, but case 2 or case 3 aren't very informative for other developers (or even you a couple of weeks later)
let's switch and sort depending on the header name (salary, age, etc)